docs/changelog/2024/2412.md

```diff
@@ -0,0 +1,157 @@
+---
+  title: Cycle 2412 (28 Nov)
+---
+
+--8<-- "includes/abbreviations.md"
+
+Cycle 12 is always a major update for NZZC, with this year being no different. Cycle 2412 implements Auckland DMAPS, ensuring further separation between departures and missed approach paths. There are also a plethora of changes to IFR waypoints.
+
+## SOP Changes
+
+### Content
+
+The following content has been added:
+
+- Nelson Aerodrome
+- Queenstown TMA
+- Client Guides for vatSys and Euroscope
+
+### Aerodromes
+
+#### NZAA: Auckland
```

Comment:
Also probably worth calling out the main waypoint changes as a pointer for people to take a closer look?

Reply:
Already highlighted under "Data Changes"